Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Build the manager binary
 # For more details and updates, refer to
 # https://catalog.redhat.com/software/containers/ubi9/go-toolset/61e5c00b4ec9945c18787690
-FROM registry.access.redhat.com/ubi9/go-toolset:1.20.10 AS builder
```

Comment:
Why go version bump is needed for removal of a feaure?

Reply:
Not really needed, but since some operators are already updated, we need to update build service too.
I agree, that it would be good to have a separate PR for that, but since we are time constrained, I did it on the go.

Reply:
I fully disagree that time constrains are reason to mix multiple things into one commit